src/seedcase_sprout/core/sprout_checks/check_data_types.py

```diff
@@ -0,0 +1,122 @@
+import polars as pl
+
+from seedcase_sprout.core.get_nested_attr import get_nested_attr
+from seedcase_sprout.core.properties import (
+    FieldProperties,
+    ResourceProperties,
+)
+from seedcase_sprout.core.sprout_checks.check_column_data_types import (
+    FRICTIONLESS_TO_COLUMN_CHECK,
+)
+from seedcase_sprout.core.sprout_checks.get_field_error_message import (
+    get_field_error_message,
+)
+
+# Column name for column containing the row index
+INDEX_COLUMN = "__row_index__"
+# Column name for column containing the result of the check
+CHECK_COLUMN_NAME = "{column}_correct"
```

Comment:
```suggestion
CHECK_COLUMN_NAME = "__{column}_correct__"
```
To decrease the chances of the check column clashing with an existing column (same as for `INDEX_COLUMN`)

Reply:
Could also be something like: 
```suggestion
CHECK_COLUMN_NAME = "{column}_check_data_types_correct"
```